### ISSUE TITLE
Extension's Update Notification Bug

### ISSUE THREAD

**Issue by bkrucarci** [kkkrist/chromium-notifier#46]
I have a few extension installed on my Chromium (win64-stable-ungoogled-marmaduke)

I also have [Chromium Web Store](https://github.com/NeverDecaf/chromium-web-store) installed.

I have the latest version 1.4.4.3 of this extension but Chromium Notifier tells me "There is an update v1.4.4.1" ??
It is old but there is always "New" notification on the Chromium Notifier icon.

Please check this.

![Screenshot_1](https://user-images.githubusercontent.com/15933805/170858594-81707501-5bf7-47b0-953b-15cccb5fed08.png)

**Comment by kkkrist** [comment]
That's because the `update_url` referenced in said extension's manifest still states 1.4.4.1 as latest version:
 
https://raw.githubusercontent.com/NeverDecaf/chromium-web-store/master/updates.xml

With this version scheme it would be easy to determine whether it's actually new or not. But it's impossible to reliably derive version order from *all* the different schemes out there (some just use hashes). So to keep it simple, a version is always reported as new when it's different.

TLDR; It's not a bug, it's just that the author of Chrome Web Store needs to update their repo.

**Comment by bkrucarci** [comment]
> That's because the `update_url` referenced in said extension's manifest still states 1.4.4.1 as latest version:
> 
> https://raw.githubusercontent.com/NeverDecaf/chromium-web-store/master/updates.xml
> 
> With this version scheme it would be easy to determine whether it's actually new or not. But it's impossible to reliably derive version order from _all_ the different schemes out there (some just use hashes). So to keep it simple, a version is always reported as new when it's different.
> 
> TLDR; It's not a bug, it's just that the author of Chrome Web Store needs to update their repo.

I read all :) No need TLDR.

I will report it to them. Thanks 👍🏻
